compensating-transactions/pom.xml

```diff
@@ -36,7 +36,11 @@
   <modules>
     <module>non-transactional_resource</module>
     <module>travel-agent</module>
-    <module>mongodb-simple</module>
+<!--     mongodb-simple quickstart is fully working,  -->
+<!--     but it will be disabled until  -->
+<!--     narayana CI infrastructure will 
+         not be able to manage this docker quickstart  -->
```

Comment:
The word `not` changes the meaning of this sentence.

Something similar to the following might be better:

```
The mongodb-simple quickstart works but requires docker to run MongoDB.
We have disabled it because our Jenkins infrastructure does not have docker installed.
But you can run the quickstart by following the instructions in the [README](README.md).
```